include/flatbuffers/flexbuffers.h

```diff
@@ -1198,6 +1199,8 @@ class Builder FLATBUFFERS_FINAL_CLASS {
 
   template<typename T> void Add(const std::vector<T> &vec) { Vector(vec); }
 
+  // Following helper overload is not for writing Blob(const void*, size_t);
```

Comment:
This is a very specific comment that is not useful to the majority of users of this very generic function. I am suspecting you had trouble writing a blob, but this is not the way to document this. The fact that you cannot pass `(const void*, size_t)` to a `T &` is probably already clear.

Reply:
i got into the problem because the blob i have in my application is of const char* type and not const void* type.. 
so while blindly using this add api for writing everything, i fell into this pitfall of writing blob wrongly..
struct Blob
{
const char* data{};  size_t size{};
}
Add(2);
Add(2.3);
Add(blob.data, blob.size); // and Bamm.. here i fell into issues!

This multi input Add api is creating confusion at client end